package.json

```diff
@@ -22,9 +14,7 @@
     "start": "react-scripts start",
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
```

Comment:
i test in futuro ci serviranno eh xD 
e' la prossima cosa che dovresti imparare secondo me :P